Public/Assets/Js/reboot.js

```diff
@@ -726,3 +726,50 @@ function disableCheckboxGroupe(checkbox,selectId) {
     }
 }
 
+function searchLdapUser() {
+    var form = document.getElementById("manageUser");
+    var nom = form.new_nom.value;
+    if(2 >= nom.length) {
+        return;
+    }
+    var page = 'hr_recherche_ldap.php?nom=' + nom;
+
+    $.ajax({
+        type : 'GET',
+        url : page,
+        dataType : 'text',
+        success : function(data)
+        {
+            var arr = new Array();
+            arr = JSON.parse(data);
+            var list = document.getElementById("suggestions");
+            var frag = document.createDocumentFragment();
+            for (var i = 0; i < arr.length; ++i) {
+                var word = document.createElement("li");
+                frag.appendChild(word);
+                word.login = arr[i]["login"];
+                word.nom = arr[i]["nom"];
+                word.prenom  = arr[i]["prenom"];
+                word.innerHTML = word.nom + " " + word.prenom;
+                word.onmousedown = function(){
+                    document.getElementById("manageUser").new_nom.focus();
```

Comment:
Là encore, je pense que le `getElementById` est consommateur, autant le factoriser (il ne changera pas dans l'itération).